src/http-api/resources/version.js

```diff
@@ -0,0 +1,14 @@
+var ipfs = require('./../index.js').ipfs
+var boom = require('boom')
+
+exports = module.exports
```

Comment:
not in this case, but it is kind of the same why I always add `{}` to if statements, keeping the code uniform, avoid having to debug someday cause 'I forgot one' or something.